umbraco-heartcore/release-notes/core-dependency-upgrade.md

```diff
@@ -28,15 +28,19 @@ After the migration ***some*** non-ASCII characters in content URLs will be subs
 **This is a breaking change.** Unless your application has been built with redirection in mind, then external links to affected content items may no longer work post-migration.
 {% endhint %}
 
-For a non-exhaustive example, see the below table to see how some character substitutions will (or won't) change:
+The below table shows a comparison of new and old behaviour for all changes. This is not a list of all characters that are substituted, rather it is a list of those that are changing behaviour only.
```

Comment:
A suggestion for a slight rewrite, as well as using US English instead of UK.
Let me know what you think:

```suggestion
The following table compares new and old behavior for all changes. This is not a list of all characters that are substituted, rather it is a list of those with changed behavior.
```

Reply:
Looks good to me!